src/plugin.js

```diff
@@ -1,12 +1,11 @@
 /* eslint import/namespace: ['error', { allowComputed: true }]*/
-import Vue from 'vue';
 
 import * as widgets from './widgets';
 
-export const plugin = Object.assign({}, widgets, {
-  install() {
+export const plugin = {
+  install(Vue) {
```

Comment:
```suggestion
  install(localVue) {
```

We could rename it to indicate that it's clearly not the global one, it avoids a bit of confusion.